libvast/src/value_index.cpp

```diff
@@ -246,15 +246,15 @@ string_index::lookup_impl(relational_operator op, const data& x) const {
         return result;
       }
       if (str_size > chars_.size())
-        return bitmap{length_.size(), op == not_equal};
+        return ids{length_.size(), op == not_equal};
```

Comment:
This is where the transition takes place: internally, the function uses bitmaps. But from the caller's perspective, it's just a set of IDs. So the function body should keep using `bitmap`s.

For the future: once `ids` is no longer a type alias, we will have a (non-explicit) constructor from `bitmap` so that this continues to work.